#!/usr/bin/env python

PACKAGE = "aiimooc_syz"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ang_res_x", double_t, 0, "resolution in  x-direction", 0.03, 0.05, 10)
gen.add("ang_res_y", double_t, 0, "resolution in  y-direction", 0.28, 0.05, 10)
gen.add("max_ang_w", double_t, 0, "angle bounds of the sensor in width", 81.7, 0, 81.7)
gen.add("max_ang_h", double_t, 0, "angle bounds of the sensor in height", 25.1, 0, 25.1)
gen.add("theta_ViewPort",double_t,0,"angle of convert PointCloud to RangeImage",0,0,81.7)

exit(gen.generate(PACKAGE, "range_image", "RangeImage"))
